addons/web/static/src/js/services/config.js

```diff
@@ -14,6 +14,11 @@ const bus = new Bus();
  * this file someday.
  */
 
+const maxTouchPoints = navigator.maxTouchPoints || 1;
+const isAndroid = /Android/i.test(navigator.userAgent);
+const isIOS = /(iPad|iPhone|iPod)/i.test(navigator.userAgent) || (navigator.platform === 'MacIntel' && maxTouchPoints > 1);
+const isOtherMobileDevice = /(webOS|BlackBerry|Windows Phone)/i.test(navigator.userAgent);
```

Comment:
I don't think we should keep a regex for this (or for anything in general), this can easily and more efficiently be done with simple `includes`.

Reply:
Note that those regexes are case-insensitive ; you would have to normalize the case of the user agent before comparing it...
Not sure about the actual performance impact ?

Reply:
Maybe something like this: ["iPad", "iPhone", "iPod"].some(el => navigator.userAgent.includes(el)); xD

Reply:
After some testing apparently Regex is significantly faster on Firefox and includes is significantly faster on Chrome...
```js
const userAgent = navigator.userAgent.toLowerCase();
const isAndroid = userAgent.includes('android');
const isIOS = ['ipad', 'iphone', 'ipod'].some(s => userAgent.includes('s')) || (navigator.platform === 'MacIntel' && maxTouchPoints > 1);
const isOtherMobileDevice = ['webos', 'blackberry', 'windows phone'].some(s => userAgent.includes('s'));
```

Reply:
But are 2 calls to some() + 6 callbacks + 6 includes() really faster than 2 regexes' test() calls?
Do we even care (in this context)? :eyes:

Reply:
Edit: I had a typo in code, in Chrome includes still faster after fixing the typo, but not by much.

So at the end, do we care? Yes if the difference was significant, it was at least worth double checking.
But in turns out it is not significant in this case, so the suggestion can be ignored.